_episodes/06-agile-in-research.md

```diff
@@ -1,33 +1,34 @@
 ---
 title: "Challenges for Agile way of working in research"
-teaching: 15
-exercises: 20
+teaching: 40
+exercises: 0
 questions:
-- "What are challenges for an agile way of working in research and how do I solve them?"
+- "What are challenges for an Agile way of working in research and how do I solve them?"
 objectives:
-- "Know how to do waterfall projects in an Agile way"
-- "Know how to deal with different expertise in a team"
-- "Remember to fit the methodology to your way of working"
-- "Know how to deal with uncertainty and dependencies in projects"
+- "List challenges for an Agile way of working in research"
+- "Know the solutions to challenges for an Agile way of working in research"
```

Comment:
I think here it makes more sense to keep it generic instead of going into the different challenges.